src/bundles/comms.stanza

```diff
@@ -111,10 +111,23 @@ public pcb-bundle can-logical :
 ; SPI - Serial Peripheral Interface
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+doc: \<DOC>
+SPI Bundle Pin Names
+A list of possible names for the SPI interface. Note that in order to connect
+SPI interfaces automatically, the constituent pins need to be present on all
+objects to be connected. 
+<DOC>
+
 public pcb-enum jsl/bundles/comms/SPIPins :
   SPI-CS
   SPI-MISO
   SPI-MOSI
+  SPI-COPI
+  SPI-CIPO
+  SPI-RDY
```

Comment:
This signal is not standard SPI.  It is a custom extension for whichever MCU we found it in - right ? 

I think this is another case for the concept of inheritance in bundles. 

Something feels wrong about putting this here. I can't put my finger on it. But if I had bundle inheritance - I would definitely not put this pin definition here.

The reason to contemplate it at all is because of the complexity of this bundle. There are several different configuration variants - the only constant is `SCLK`.

The only other option I can think to do is to create helper "generator" functions that construct the internals of an SPI bundle consistently with options and then allow the user to add their custom port definitions in their derived SPI bundles.

@CuppoJava - FYI

Reply:
Yeah, let's chat about this.

Reply:
Removing extra pins, would love to find a way to do bundle inheritance as well to extend the base bundle for vendor-specific extensions.